src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -409,6 +407,6 @@ internal void VerifyWrite(int originalDepth, Utf8JsonWriter writer)
         /// <param name="writer">The <see cref="Utf8JsonWriter"/> to write to.</param>
         /// <param name="value">The value to convert.</param>
         /// <param name="options">The <see cref="JsonSerializerOptions"/> being used.</param>
-        public abstract void Write(Utf8JsonWriter writer, [DisallowNull] T value, JsonSerializerOptions options);
+        public abstract void Write(Utf8JsonWriter writer, T value, JsonSerializerOptions options);
```

Comment:
Do we have tests that pass in null?

Reply:
We have some for for cases where the type to convert is a struct - https://github.com/dotnet/runtime/blob/cc132d790af6a5971925ef476d9411b251e91cfe/src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests.NullValueType.cs.

Reply:
Thanks.  It'd be good to have null tests for both value types and reference types.

Reply:
> Do we have tests that pass in null successfully?

Null strings use this path.

Reply:
Is there any special guidance for using these attributes on abstract or non-sealed types\members? A derived class, for example, may want\need different behavior.

Is it OK to place [DisallowNull] on an abstract base class but then still allow null and use a nullable Type (like we do in the String converter).